zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/service/exceptions/SessionsVerificationException.java

```diff
@@ -1,6 +1,6 @@
-package com.github.onsdigital.zebedee.session.store.exceptions;
+package com.github.onsdigital.zebedee.session.service.exceptions;
```

Comment:
Ditto

Reply:
Removed this as well among a couple others to which this applies.